ledger/participant-integration-api/src/main/scala/platform/apiserver/error/LedgerApiErrors.scala

```diff
@@ -0,0 +1,506 @@
+// Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.platform.apiserver.error
+
+import com.daml.error._
+import com.daml.lf.data.Ref
+import com.daml.lf.data.Ref.PackageId
+import com.daml.lf.engine.Error.Validation.ReplayMismatch
+import com.daml.lf.engine.{Error => LfError}
+import com.daml.lf.interpretation.{Error => LfInterpretationError}
+import com.daml.lf.language.{LanguageVersion, LookupError, Reference}
+import com.daml.lf.transaction.GlobalKey
+import com.daml.lf.value.Value
+import com.daml.lf.{VersionRange, language}
+import com.daml.logging.{ContextualizedLogger, LoggingContext}
+import com.daml.platform.apiserver.error.ErrorGroups.ParticipantErrorGroup.TransactionErrorGroup.LedgerApiErrorGroup
+
+object LedgerApiErrors extends LedgerApiErrorGroup {
+
+  // the authorization checks are here only for documentation purpose.
+  object AuthorizationChecks extends ErrorGroup() {
+
+    @Explanation(
+      """This rejection is given if the submitted command does not contain a JWT token on a participant enforcing JWT authentication."""
+    )
+    @Resolution("Ask your participant operator to provide you with an appropriate JWT token.")
+    object Unauthenticated
+        extends ErrorCode(
+          id = "UNAUTHENTICATED",
```

Comment:
maybe add a todo. we haven't yet threaded through these errors in the auth token handlers. i.e. these errors are not yet used anywhere.

Reply:
Done